plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -712,7 +712,9 @@ private Optional<View> getIcebergView(ConnectorSession session, SchemaTableName
             return Optional.empty();
         }
         catch (RESTException e) {
-            throw new TrinoException(ICEBERG_CATALOG_ERROR, "Failed to load view '%s'".formatted(viewName.getTableName()), e);
+            throw new TrinoException(ICEBERG_CATALOG_ERROR,
+                    "Failed to load view '%s', the configuration 'iceberg.rest-catalog.view-endpoints-enabled' is '%s'".formatted(viewName.getTableName(),
```

Comment:
This change doesn't make sense to me. We should include the property name only when the error cause is related to the property.